agents1/OfficialAgent.py

```diff
@@ -250,8 +252,10 @@ def decide_on_actions(self, state):
                     self._sendMessages = []
                     self.received_messages = []
                     self.received_messages_content = []
-                    self._sendMessage('Going to re-search all areas.', 'RescueBot')
+                    value_to_decrease = 0.2
+                    self._sendMessage('Going to re-search all areas. Willingness will decrease by' + str(value_to_decrease) + 'as a consequence.', 'RescueBot')
```

Comment:
```suggestion
                    self._sendMessage(f'Going to re-search all areas. Willingness will decrease by {str(value_to_decrease)} as a consequence.', 'RescueBot')
```
Spacing was a bit messed up